### ISSUE TITLE
unchecked radiobuttons wrong sensitive/insensitive look

### ISSUE THREAD

**Issue by theoreffet** [B00merang-Project/Windows-10#165]
Hello, I have noticed in a gtk2 application that the unchecked insensitive radiobuttons look perfectly sensitive.
The insensitive ones have black outline while the sensitive ones have grayed-out outline, I would have expected the opposite behavior.
After a closer look at the source pictures it seems that this behavior is not consistent between the checked/unchecked/mixed radiobuttons. Neither is it consistent between the gtk2/gtk3/gtk3.2 sources. the gtk3.2 assets are like I would have expected.

To sum it up I think the following picture in the gtk2 assets should be swapped:
"radio-unchecked.png" <-> "radio-unchecked-insensitive.png"
And in the gtk3 assets:
"radio-unselected.png" <-> "radio-unselected-insensitive.png"
"radio-unselected@2.png" <-> "radio-unselected-insensitive@2.png"

As far as I have seen, there is the same issue in the Windows-10-dark theme.

I guess this should be pretty straightforward to fix but let me know if I should start a pull-request anyway. Thanks!

**Comment by Elbullazul** [comment]
we redid the asset generation process this summer and rebased them on the source material. I hadn't noticed the difference between GTK2/3 and 3.20 assets

if you can replace the gtk2/3 assets with those from the gtk-3.20 (respecting filenames) and submit a PR it will be much appreciated, as I will be busy for the next couple of weeks
